stdlib/Statistics/src/Statistics.jl

```diff
@@ -74,6 +74,7 @@ function mean(f, itr)
     return total/count
 end
 mean(f, A::AbstractArray) = sum(f, A) / length(A)
+mean(f, A; dims = 1) = sum(f, A, dims = dims) / size(A, dims)
```

Comment:
```suggestion
mean(f, A::AbstractArray; dims = 1) = sum(f, A, dims = dims) / size(A, dims)
```
Maybe better put it below with the other method taking `dims`, which makes this type condition clearer.

Reply:
Also a doctstring should mention this (probably the one with `dims`).

Reply:
I put this near the one with `Function` argument and modified the docstring accordingly.